ui/src/menu.test.tsx

```diff
@@ -0,0 +1,36 @@
+import Button from "@mui/material/Button";
+import MenuItem from "@mui/material/MenuItem";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { useMenu } from "./menu";
+
+function MenuButton(props: { onClick: () => void }) {
+  const [menu, show] = useMenu({
+    children: [
+      <MenuItem key="1" onClick={props.onClick}>
```

Comment:
since you only have one child, I think you can drop the [], pass <MenuItem>...</MenuItem directly to children, and remove the key.

Reply:
Done